testutil/syncer.go

```diff
@@ -7,15 +7,15 @@ import (
 	"go.sia.tech/coreutils/syncer"
 )
 
-// A MemPeerStore is an in-memory implementation of a PeerStore.
-type MemPeerStore struct {
+// A EphemeralPeerStore is an in-memory implementation of a PeerStore.
+type EphemeralPeerStore struct {
```

Comment:
Not sure how strict we want to be just yet but `coreutils` is a library and this is breaking part of the libraries interface which makes it a breaking change and not a `minor`.

Reply:
Breaking would push this to v1.0.0, which I don't think we're ready for quite yet.

Reply:
just me being pedantic about consistent names. Probably not worth the breakage